File: backend/InsuranceAPI/Models/Phone.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace InsuranceAPI.Models;

public partial class Phone
{
    public long Id { get; set; }

    public long Insured { get; set; }

    public string Number { get; set; } = null!;

    public string? Description { get; set; }

    [JsonIgnore]
    public virtual Insured InsuredNavigation { get; set; } = null!;
}

File: src/app/models/Phone.ts
export class Phone{
    public number:string;
    public insured?:number;
    public id?:number;
    public description?:string;

    constructor(number:string,description?:string,insured?:number,id?:number,){
        this.insured = insured;
        this.number = number;
        this.id = id;
        this.description = description;
    }
}